optimum/gptq/quantizer.py

```diff
@@ -450,6 +564,8 @@ def store_input_hook(_, input, *args):
                     raise ValueError(f"Module {module_name} was not found in model")
 
         torch.cuda.empty_cache()
+        if hasattr(torch, "xpu"):
```

Comment:
not sure but I have seen it multiple times don't we have to also check `and torch.xpu.is_available()` ?

Reply:
done